docs/cloud/managing-airbyte-cloud/manage-data-residency.md

```diff
@@ -8,7 +8,7 @@ Default data residency allows you to choose where your data is processed.
 
 :::note 
 
-Configuring default data residency only applies to new connections and does not affect existing connections.   
+To ensure your data is processed in the correct region, set your default data residency before creating a new connection as some workflows rely on the default data residency. When you set the default data residency, it applies to new connections and does not affect existing ones. If you do not set the default data residency, the [Airbyte Default](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud/#united-states-and-airbyte-default) region is used. 
```

Comment:
Looks good! a minor comment though:

Should we be more specific, such as mentioning some workflows during **connection creation** will rely on the data residency, maybe even giving examples such as workflow for fetching data schema or checking connection could be setup, so user could understand why it's important to setup their data residency before creating a connection? (just my 2 cents, feel free to push back if you think this is too much detailed information for users)

Reply:
I agree with Xiaohan here: specifically we could say "testing sources and destinations and fetching schema rely on the default data ..." instead of "some workflows"

Reply:
Thank you for the feedback! I updated the text with info about testing sources and destinations and fetching the schema :)